src/components/MovieList/MovieList.js

```diff
@@ -1,7 +1,7 @@
 import './MovieList.css';
 import mockMovieData from '../../mockData/mockMovieData';
 import MovieCard from '../MovieCard/MovieCard';
-import ErrorBoundary from '../errorBoundary/ErrorBoundary';
+import ErrorBoundary from '../ErrorBoundary/ErrorBoundary';
```

Comment:
[Sin](https://stackoverflow.com/questions/11183788/in-a-git-repository-how-to-properly-rename-a-directory)